Automattic/jetpack
# My Jetpack: Add product/Boost interstitial component

This PR creates a couple of interstitial components, making the focus on Boost.

![image](https://user-images.githubusercontent.com/77539/152362930-c6b22ab5-3be5-4a5b-b791-d933867f0fb8.png)

We can keep iterating in follow-up PRs.

Fixes https://github.com/Automattic/jetpack/issues/22572

#### Changes proposed in this Pull Request:

* My Jetpack: Add product/Boost interstitial component 

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






* Build My Jetpack

```bash
> jetpack build packages/my-jetpack
```

* Run storybook

```bash
cd ./jetpack/projects/js-packages/storybook
```

```bash
> pnpm install & pnpm storybook:dev
```

Check the different My Jetpack components from there

https://user-images.githubusercontent.com/77539/152392413-0d6a0315-8233-4224-8608-8525ff71b732.mov
